client/index.js

```diff
@@ -1,2 +1,8 @@
-import './view/base/index.js'
-import './view/browsers/index.js'
+import { initForm } from './view/Form/form.js';
+import { initQueryLinks } from './view/QueryLink/queryLink.js';
+
+initForm();
```

Comment:
Very strange. I have ESLint rule to not adding `;`

Reply:
Mmm... now there is just extend of your config.
<img width="796" alt="image" src="https://user-images.githubusercontent.com/27442574/183066280-add29c7a-e1a7-47d6-9ade-7337c6be86f9.png">

Reply:
The code style formatting is done by Prettier, not ESLint

Reply:
(We can add Prettier to ESLint, but Prettier adds 100 MB to `node_modules`, it is better to keep it in text editor)

Reply:
This command remove symdol only from index.js, is it what you want?
<img width="705" alt="image" src="https://user-images.githubusercontent.com/27442574/183093386-4f9c1f15-1f45-4750-85b5-3c10244cfc8f.png">

Reply:
Yes, the project’s code style is like it.

Run also `npx prettier client/*/*.js` to format other JS files (maybe we need to add some other files to the call too).

Reply:
Yes, that is what i meant) Will be fixed in a minute)